Android Setup: Add information about Java and Gradle installation (#772)
fastlane/docs: docs/getting-started/android/setup.md

## Setting up _fastlane_

> Android Studio maintains its own instances of Gradle and Java, which you do not have access to in the command line, so even though you can build and run apps in Android Studio, you need to install those tools for fastlane. Additionally, you need to install Java 8 and not later versions of Java, which do not play well with Gradle.

Reviewer: ```suggestion
Android Studio maintains its own instances of Gradle and Java, which you do not have access to in the command line. So even though you can build and run apps in Android Studio, you need to install those tools for fastlane separately. Additionally, you need to install Java _8_ and not a later version of Java, which do not play well with Gradle.
```

### Install Java 8
For MacOS, the easiest way to install Java 8 is with [Homebrew](https://brew.sh/).

Run the following commands in the terminal:
`brew tap caskroom/versions`
`brew cask install java8`

> The terminal command: `javac -version` should now return `javac 1.8.0_202-ea`

Reviewer: Exactly this version or similar?

Author: Something similar. I'll update

> If this is not the case, your JAVA_HOME environment variable likely refers to a more recent version of Java. The easiest way to fix this is to uninstall all java instances via Homebrew with `brew cask uninstall java` and then install java 8 with `brew cask install java8`

Reviewer: ```suggestion
If this is not the case, your `JAVA_HOME` environment variable likely refers to a more recent version of Java. The easiest way to fix this is to uninstall all Java instances via Homebrew with `brew cask uninstall java` and then install Java 8 with `brew cask install java8`.
```

Can you actually uninstall Java versions not installed via Homebrew with it?

Author: I installed the latest Java via Homebrew before I discovered that it was the incorrect version and then needed to uninstall it. I imagine you can only uninstall the Homebrew Java versions.

### Install Gradle

> on MacOs you can install Gradle via homebrew with the command `brew install gradle`

Reviewer: ```suggestion
On macOS you can install Gradle via homebrew with the command `brew install gradle`.
```

## Setting up _fastlane_

Android Studio maintains its own instances of Gradle and Java, which you do not have access to in the command line. So even though you can build and run apps in Android Studio, you need to install those tools for fastlane separately. Additionally, you need to install Java _8_ and not a later version of Java, which do not play well with Gradle.

### Install Java 8
For macOS, the easiest way to install Java 8 is with [Homebrew](https://brew.sh/).

Run the following commands in the terminal:
```
brew tap caskroom/versions
brew cask install java8
```

The terminal command: `javac -version` should now return something similar to: `javac 1.8.0_202-ea`

> If this is not the case, your JAVA_HOME environment variable likely refers to a more recent version of Java. The easiest way to fix this is to uninstall all Java instances which were installed via Homebrew with `brew cask uninstall java` and then install Java 8 with `brew cask install java8`

Reviewer: ```suggestion
If this is not the case, your `JAVA_HOME` environment variable likely refers to a more recent version of Java. The easiest way to fix this is to uninstall all Java instances which were installed via Homebrew with `brew cask uninstall java` and then install Java 8 with `brew cask install java8`
```
